src/Symfony/Bundle/SecurityBundle/Tests/Functional/Bundle/CsrfFormLoginBundle/Form/UserLoginFormType.php

```diff
@@ -29,18 +29,15 @@
  */
 class UserLoginFormType extends AbstractType
 {
-    private $request;
+    private $requestStack;
 
-    /**
-     * @param Request $request A request instance
-     */
-    public function __construct(Request $request)
+    public function __construct(RequestStack $requestStack)
```

Comment:
This change could even be done in older branches to simplify merging between branches, given it is in our functional tests.